website/templates/project/settings.mako

```diff
@@ -306,32 +306,39 @@
                     <div class="panel-body">
                         % if not node['institution']['name']:
                             <div data-bind="visible: !loading()">
-                                <div data-bind="visible: availableInstitutions().length">
+                                <div data-bind="visible: error()">
                                     <div class="help-block">
-                                        Projects affiliated with institutions will show some institutional branding (such as logos) and if public, will be discoverable on OSF institutional landing pages.
-
-                                        You are authorized to affiliate your projects with the following institutions:
+                                        Failed to load up your institutions. Please try again.
```

Comment:
Keep your error messages consistent with the rest of the site. For example, the message should include a mailto link to the support email.